api/v2/db/ldap/connection.js

```diff
@@ -0,0 +1,28 @@
+const config = require('config');
+const ldap = require('ldapjs');
+
+const ldapConfig = config.get('dataSources').ldap;
+
+/**
+ * @summary Get an ldap connection
+ * @function
+ * @returns ldapjs client connection object
```

Comment:
Can you also describe the returns type like here? https://github.com/osu-mist/notes-api/blob/master/api/v1/db/json/fs-operations.js#L30